Proton/Sources/Editor/EditorView.swift

```diff
@@ -877,41 +880,59 @@ extension EditorView: RichTextViewDelegate {
 }
 
 extension EditorView {
-    func invalidateLayout(for range: NSRange) {
-        richTextView.invalidateLayout(for: range)
-    }
-
-    func relayoutAttachments(in range: NSRange? = nil) {
```

Comment:
Changed the API here.

Instead of `NSRange?` where:
- `nil` means “all attachments” 
- non-nil means “If there is an attachment in that range, lay it out” (which can means the caller has to get the range and pass it in, and could pass in the wrong range).

Now, there are two discrete calls:
- `relayoutAllAttachments()`
- `relayoutAttachment(_:)`

Both of them call out to `relayout(attachment:range:)` which is private.